hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/AbstractTransaction.java

```diff
@@ -289,8 +289,9 @@ protected void commitOrRollback() {
                 LOG.error("Failed to rollback changes:\n {}", mutation, e2);
             }
             // Rethrow the commit exception
+            // The e.getMessage maybe too long to see key information
```

Comment:
The e.getMessage maybe too long to see key information, therefore use e.getCause